root/edit/details/AddWork.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import commaOnlyList from '../../static/scripts/common/i18n/commaOnlyList';
+
+
+ type AddWorkEditProps = {
+   ...EditT,
+   +display_data: {
+     +attributes?: AttributeT,
+     +comment?: string,
+     +iswc?: string,
+     +language?: LanguageT,
+     +languages: $ReadOnlyArray<string>,
```

Comment:
That's included conditionally, like `language`. See `build_display_data`.
```suggestion
     +languages?: $ReadOnlyArray<string>,
```

Reply:
Incase of this, I tried creating component with no lyrics language, the output was an empty array in the console log

Reply:
That's only for new edits, though. There are thousands of old edits that only have `language_id` in the data, e.g. https://musicbrainz.org/edit/17725386/data or even older ones that have neither, e.g. http://musicbrainz.org/edit/14459931/data

Reply:
ok